src/eko/io/types.py

```diff
@@ -63,6 +65,14 @@ class CouplingsRef(DictLike):
 
     alphas: FloatRef
     alphaem: FloatRef
+    num_flavs_ref: Optional[FlavorsNumber]
```

Comment:
inside you now call it "scale" - maybe change also here?

Reply:
Impossible, "scale" is the reference scale for running quantities, so it is always a member of a `FloatRef` or an `IntRef`.

Reply:
Mmm? but it refers exactly to that "scale" - it is the number of flavors at the scale ... if you wish `ref_scale`? (a more drastic option would be to move even inside `FloatRef` - not sure we want that ...)

Reply:
The scale related to `num_flavs_ref` is the scale of `alphas`, that is already present: indeed both `FloatRef` and `IntRef` are a 2-tuple (aka a couple), whose first element is either a `float` or an `int`, and the second one is just a `float`, which is actually the scale.

So the scale of `num_flavs_ref` is `alphas.scale`, and as you can see it is already named `scale`.